src/main/java/com/qaprosoft/carina/demo/gui/components/FooterMenu.java

```diff
@@ -41,6 +41,9 @@ public FooterMenu(WebDriver driver, SearchContext searchContext) {
 
     public HomePage openHomePage() {
         homeLink.click();
+        if (newsLink.toString().equals("qwe")) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the "qwe" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://demo.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS1132)